src/plugins/kibana_react/public/field_button/field_button.tsx

```diff
@@ -68,12 +74,12 @@ export function FieldButton({
 
   return (
     <div className={classes} {...rest}>
-      <div className="kbnFieldButton__content">
+      <button onClick={onClick} className="kbnFieldButton__content">
```

Comment:
Why only include `fieldIcon` and `fieldName` into the button here?

Reply:
Because we can't have interactive content inside a button. Really, we can't have anything custom inside a button as all semantics get overwritten inside a button but this becomes most important when trying to nest interactive content.

Inside a button, everything is just "button content". You can try to fake it and it but there isn't any official support for "this part of the button does A while clicking inside of this sub-part of the button does B" so, to get around this but keeping design the same, we have a button with other interactive content layered over it and positioned with CSS.